iree/samples/emitc_modules/import_module_test.cc

```diff
@@ -0,0 +1,88 @@
+// Copyright 2020 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/base/status_cc.h"
+#include "iree/samples/emitc_modules/import_module_a.h"
+#include "iree/samples/emitc_modules/import_module_b.h"
+#include "iree/testing/gtest.h"
+#include "iree/testing/status_matchers.h"
+#include "iree/vm/api.h"
+#include "iree/vm/ref_cc.h"
+
+namespace iree {
+namespace {
+
+class VMImportModuleTest : public ::testing::Test {
+ protected:
+  virtual void SetUp() {
+    IREE_CHECK_OK(iree_vm_instance_create(iree_allocator_system(), &instance_));
+
+    iree_vm_module_t* module_a = nullptr;
```

Comment:
can you try using vm::ref<iree_vm_module_t> here? (I don't know if it works today - it may need a bit of glue to be supported)

Reply:
I couldn't get it to work, as an implicit conversion from `vm::ref<iree_module_t>*` to `iree_module_t**` doesn't exist.

Reply:
ah - we should be able to make that work - but fine for now.